packages/fx-core/resource/package.nls.json

```diff
@@ -98,24 +98,24 @@
-  "plugins.spfx.error.scaffoldError": "Project creation failed. A possible reason could be from Yeoman SharePoint Generator. Check [Output panel](%s) for details.",
-  "plugins.spfx.error.import.retrieveSolutionInfo": "Failed to retrieve existing SPFx solution information. Please make sure your SPFx solution is valid.",
-  "plugins.spfx.error.import.copySPFxSolution": "Failed to copy existing SPFx solution: %s",
-  "plugins.spfx.error.import.updateSPFxTemplate": "Failed to update project templates with existing SPFx solution: %s",
-  "plugins.spfx.error.import.common": "Failed to import existing SPFx solution to Teams Toolkit: %s",
+  "_plugins.spfx.error.installLatestDependencyError.comment": "The product brand names 'Microsoft Learn' and 'SharePoint' should not be localized.",
+  "plugins.spfx.error.scaffoldError": "Project creation is unsuccessful, which may be due to Yeoman SharePoint Generator. Check [Output panel](%s) for details.",
+  "plugins.spfx.error.import.retrieveSolutionInfo": "Unable to get existing SPFx solution information. Ensure your SPFx solution is valid.",
+  "plugins.spfx.error.import.copySPFxSolution": "Unable to copy existing SPFx solution: %s",
+  "plugins.spfx.error.import.updateSPFxTemplate": "Unable to update project templates with existing SPFx solution: %s",
+  "plugins.spfx.error.import.common": "Unable to import existing SPFx solution to Teams Toolkit: %s",
   "plugins.spfx.import.title": "Importing SPFx solution",
   "plugins.spfx.import.copyExistingSPFxSolution": "Copying existing SPFx solution...",
   "plugins.spfx.import.generateSPFxTemplates": "Generating templates based on solution info...",
   "plugins.spfx.import.updateTemplates": "Updating templates...",
-  "plugins.spfx.import.success": "Your SPFx solution has been successfully imported to %s.",
-  "plugins.spfx.import.log.success": "Teams Toolkit has imported your SPFx solution successfully. A complete log of import details can be found in %s.",
-  "plugins.spfx.import.log.fail": "Teams Toolkit failed to import your SPFx solution. A complete log of import details can be found in %s.",
-  "plugins.spfx.addWebPart.confirmInstall": "SPFx version in your solution is %s, not yet installed on your machine. Do you want to install SPFx %s in Teams Toolkit directory to continue adding web part?",
+  "plugins.spfx.import.success": "Your SPFx solution is successfully imported to %s.",
+  "plugins.spfx.import.log.success": "Teams Toolkit has successfully imported your SPFx solution. Find complete log of import details in %s.",
+  "plugins.spfx.import.log.fail": "Teams Toolkit is unable to import your SPFx solution. Find complete log of important details in %s.",
+  "plugins.spfx.addWebPart.confirmInstall": "SPFx %s version in your solution isn't installed on your machine. Do you want to install it in Teams Toolkit directory to continue adding web parts?",
```

Comment:
to confirm will this be "%s version" or "version %s"  ? I found the L116 uses "version %s".